DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/AddOns/JobProfile/Classifications/Structure/taxonomies.sf

```diff
@@ -133,11 +133,35 @@
       }
     ]
   },
+  {
+    "Id": "4697627c-2bb0-4b3f-b455-798d27048deb",
+    "Name": "JobProfileKeywords",
+    "Type": "Telerik.Sitefinity.Taxonomies.Model.FlatTaxonomy",
+    "LastModified": "2018-06-29T08:36:41.407Z",
+    "Title": [
+      {
+        "Key": "",
+        "Value": "Job Profile Keywords"
```

Comment:
This should not be there, this is a old taxon.

Reply:
Removed